components/cam/src/physics/rrtmgp/radconstants.F90

```diff
@@ -256,6 +265,42 @@ end subroutine get_sw_spectral_boundaries
 
 !------------------------------------------------------------------------------
 
+subroutine get_sw_spectral_midpoints(band_midpoints, units)
+   character(len=*), intent(in) :: units
+   real(r8), intent(out) :: band_midpoints(nswbands)
+   real(r8), dimension(nswbands) :: lower_bounds, upper_bounds
+   integer :: i
+
+   ! Get band limits
+   call get_sw_spectral_boundaries(lower_bounds, upper_bounds, units)
+
+   ! Compute band midpoints
+   band_midpoints = 0._r8
+   do i = 1,nswbands
+      band_midpoints(i) = (lower_bounds(i) + upper_bounds(i)) / 2._r8
```

Comment:
isn't is better to avoid using "._r8" here and just use "2"?

Reply:
I've recently been advised to use the explicit kind on the floating point literals, even though I think they end up being identical in these cases.